project_stage_allow_timesheet/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Duplicate key 'license' in dictionary](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3718665)